### DISCUSSION TITLE
Getting wrong ip ?

### DISCUSSION THREAD

**Discussion by shubhamjain108-lab** [chatwoot/chatwoot#2481] [Q&A] [answered]
i am running chatwoot ip address as http://172.28.128.3:9200/ and when i am creating bot it is giving wrong ip address.

    <script>
      (function(d,t) {
        var BASE_URL="http://0.0.0.0:3000";
        var g=d.createElement(t),s=d.getElementsByTagName(t)[0];
        g.src=BASE_URL+"/packs/js/sdk.js";
        s.parentNode.insertBefore(g,s);
        g.onload=function(){
          window.chatwootSDK.run({
            websiteToken: 'T197bHDnAdjfsjlflshflshljzoD',
            baseUrl: BASE_URL
          })
        }
      })(document,"script");
    </script>
    

please provide idea for config change?

**Comment by pranavrajs** [comment, marked as answer]
Please set the FRONTEND_URL to the right IP address. https://www.chatwoot.com/docs/self-hosted/configuration/environment-variables#configure-frontend-url

**Reply by shubhamjain108-lab** [reply]
working, thank you pranavrajs

**Reply by mahfuzplabon** [reply]
Hello,

I have a silly question. I installed via Docker but with 0.0.0.0 IP. Now I changed in .env file. How to reload the whole system ?? I rebooted the machine, but didn't work. Where can I find the commands to get it relaoded ?

**Reply by mahfuzplabon** [reply]
Got it via docker-compose up -d
